src/_guides/language/concurrency/index.md

```diff
@@ -360,6 +362,20 @@ The following figure summarizes the communication between
 the main isolate and the second isolate:
 
 ![PENDING: alt text here](/guides/language/concurrency/images/isolate-api.png)
+<br>
+***[PENDING: Background worker -> Worker isolate]***
+
+{{site.alert.info}}
+  **API note:**
+  When an isolate calls [`Isolate.spawn()`][],
+  it sets up the new isolate with a copy of the calling isolate’s code.
```

Comment:
@kwalrath did you by any chance see my Friday's email reply to your question regarding this?

Reply:
I was just about to get to it!

Reply:
I've just updated the doc, and I hope this version will stage successfully. ptal!